src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/NoQuic/NoQuicImplementationProvider.cs

```diff
@@ -0,0 +1,10 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Net.Quic.Implementations.NoQuic
+{
+    internal sealed class NoQuicImplementationProvider : QuicImplementationProvider
+    {
+        public override bool IsSupported => false;
```

Comment:
The linker won't be able to see this logic so the change no longer fixes #46915 issue

Reply:
> The linker won't be able to see this logic so the change no longer fixes #46915 issue.

I'll try to fix that, but I need a way how to test it and confirm it work as it should. ATM, I don't even know how the desired result looks let alone be able to confirm it.

> It will also trigger a platform compatibility warning

How can I test this?